tsconfig.json

```diff
@@ -0,0 +1,26 @@
+{
+  "compilerOptions": {
+    "module": "NodeNext",
+    "target": "ES2022",
+    "moduleResolution": "NodeNext",
+    "lib": ["ESNext"],
+    "noEmit": true,
+    "skipLibCheck": true,
+    "strict": true,
+    "declaration": true,
+    "verbatimModuleSyntax": true,
+    "esModuleInterop": true,
+    "resolveJsonModule": true,
+    "forceConsistentCasingInFileNames": true,
+    "stripInternal": true,
+    "paths": {
+      "@cloudinary-util/types": ["./packages/types/src/index.ts"],
+      "@cloudinary-util/url-loader": ["./packages/url-loader/src/index.ts"],
+      "@cloudinary-util/url-loader/schema": [
+        "./packages/url-loader/src/schema.ts"
+      ],
+      "@cloudinary-util/util": ["./packages/util/src/index.ts"]
+    }
```

Comment:
I haven't seen this needed before - is it used for anything?

Reply:
It is not needed but it ensures types across packages stay in sync during dev without having to rebuild. It also works for runtime code using a loader like `tsx` that respects tsconfig paths.

See https://colinhacks.com/essays/live-types-typescript-monorepo for more info